flytekit/clients/raw.py

```diff
@@ -110,6 +112,21 @@ def __init__(self, cfg: PlatformConfig, **kwargs):
         self._cfg = cfg
         if cfg.insecure:
             self._channel = grpc.insecure_channel(cfg.endpoint, **kwargs)
+        elif cfg.insecure_skip_verify:
+            # Get the certificate from the endpoint at port 443
+            cert = ssl.get_server_certificate((cfg.endpoint, 443))
```

Comment:
the cfg.endpoint should have the "host:port", if it doesn't have a port only then can we assume 443...

Reply:
ok, changed